packages/scripts/src/utils/constants.js

```diff
@@ -2,6 +2,7 @@
 const REQUIRED_SIGNER_ADDRESSES = {
   deployer: "0x2bAaA41d155ad8a4126184950B31F50A1513cE25",
   foundation: "0x7a3A1c2De64f20EB5e916F40D11B01C441b2A8Dc",
+  account_with_uma: "0xcb287f69707d84cbd56ab2e7a4f32390fa98120b",
```

Comment:
this EOA does not have enough to meet the gat on it's own (it's set to 5.5mm in the settings but this account only has 5mm. it might be better to "take" funds in your simulation from the `Governor` here: `0x592349F7DeDB2b75f9d4F194d4b7C16D82E507Dc` which holds the UMA protocol's treasury.